java/src/jmri/jmrix/loconet/alm/RoutesProgSingleton.java

```diff
@@ -0,0 +1,23 @@
+package jmri.jmrix.loconet.alm;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+/**
+ * Singleton used to allow, at most, one Digitrax 7th-generation Accessory
+ * Decoder "routes programmer" being instantiated.
+ *
+ * @author B. Milhaupt (C) 2024
+ */
+public enum RoutesProgSingleton {
+    INSTANCE;
```

Comment:
No need to change, but I cannot see anywhere there this class is used.

Reply:
This is used "for the next thing" I have in mind for a future pull request...